Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1908,6 +1911,109 @@ def get_user():
+        channel_id = channel.get('id')
+
+    if not channel_id:
+        return_error('Channel was not found - Either the Slack app is not a member of the channel, '
+                     'or the slack app does not have permission to find the channel.')
+    if not thread_id:
+        return_error('The timestamp of the message to edit is required.')
+
+    if message and not blocks:
+        if ignore_add_url and isinstance(ignore_add_url, str):
+            ignore_add_url = bool(strtobool(ignore_add_url))
+        if not ignore_add_url:
+            investigation = demisto.investigation()
+            server_links = demisto.demistoUrls()
+            if investigation:
+                if investigation.get('type') != PLAYGROUND_INVESTIGATION_TYPE:
+                    link = server_links.get('warRoom')
+                    if link:
+                        if entry:
+                            link += '/' + entry
+                        message += f'\nView it on: {link}'
+                else:
+                    link = server_links.get('server', '')
+                    if link:
+                        message += f'\nView it on: {link}#/home'
+
+    body = {
+        'channel': channel_id,
+        'ts': thread_id
+    }
+    if message:
+        clean_message = handle_tags_in_message_sync(message)
+        body['text'] = clean_message
+    if blocks:
+        block_list = json.loads(blocks, strict=False)
+        body['blocks'] = block_list
+    try:
+        response = send_slack_request_sync(CLIENT, 'chat.update', body=body)
+
+        hr = "The message was successfully edited."
+        result_edit = {
+            'ID': response.get('ts', None),
+            'Channel': response.get('channel', None),
+            'Text': response.get('text', None)
+        }
+        context = {
+            'Slack.Thread(val.ID === obj.ID)': result_edit
+        }
+        return_results(CommandResults(
+            readable_output=hr,
+            outputs=context,
+            raw_response=json.dumps(response.data)))
+
+    except SlackApiError as slack_error:
+        return_error(f"{slack_error}")
+
+
+def pin_message():
+    channel = demisto.args().get('channel')
+    thread_id = demisto.args().get('threadID')
+
+    channel_id = ''
+
+    if not channel:
+        mirror = find_mirror_by_investigation()
+        if mirror:
+            channel_id = mirror['channel_id']
+    else:
+        channel = get_conversation_by_name(channel)
+        channel_id = channel.get('id')
+
+    if not channel_id:
+        return_error('Channel was not found - Either the Slack app is not a member of the channel, '
+                     'or the slack app does not have permission to find the channel.')
+    body = {
+        'channel': channel_id,
+        'timestamp': thread_id
+    }
+    try:
+        send_slack_request_sync(CLIENT, 'pins.add', body=body)
+        return_results('The message was successfully pinned.')
+
+    except SlackApiError as slack_error:
```

Comment:
how does the bad thread id error looks?

Reply:
![Screen Shot 2021-08-23 at 17 06 27](https://user-images.githubusercontent.com/42912128/130461432-d5b627f7-3186-4886-a678-be327ba36fe9.png)
Like this. When in debug mode, it will output the traceback as well.